HandsOnProject/app/src/main/java/com/example/handsonproject/MainActivity.kt

```diff
@@ -1,11 +1,144 @@
 package com.example.handsonproject
 
-import androidx.appcompat.app.AppCompatActivity
+
 import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.example.handsonproject.databinding.ActivityMainBinding
+import com.example.handsonproject.databinding.ActivityMainBinding.inflate
+
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+    private var op : Int = 0
```

Comment:
@shovian It would be better if this was an enum, not an integer. Always prefer enums in such cases. It helps with readability, is easier to debug and prevents plenty of errors to begin with.

Reply:
Looks much better now!